### ISSUE TITLE
Use JSONAPI standard

### ISSUE THREAD

**Issue by gschorkopf** [atlanta-ember-org/atlanta-ember-backend#10]
AFAIK this repository uses REST, and not the jsonapi.org standard. Am I wrong? Should we consider moving in this direction? I feel like https://github.com/cerebris/jsonapi-resources is the gold standard here.

**Comment by pdougall1** [comment]
I agree @gschorkopf, but imagine that would require a significant refactor?  I just made a PR #15  that addresses annoying things about conforming to that standard for error messages.  That makes me thing we'll end up doing more work to conform to it than the refactoring to just use jsonapi-resources would be.  You have experience here so would love your opinion on whether reworking to use https://github.com/cerebris/jsonapi-resources is a worthwhile endeavor.

**Comment by gschorkopf** [comment]
It wouldn't be significant if we nip it in the bud now now. Error messaging is pretty simple in JSONAPI land. 

Honestly, the main thing is: we're Ember devs. `JSONAPIAdapter` is a first class citizen in Ember. And though it isn't right now in Rails (Active Model Serializers, which even in the README it's highly suggested you use JSONAPI https://github.com/rails-api/active_model_serializers#about) We should treat it as such. 

I know it'll require pain the the short term, but believe it's worth it if we actually want to build a product that is truly "convention over configuration" ala Rails/Ember. Plus, jsonapi-resources is a jam, and makes life super simple for the API.

I'm on vacation. Will try to address this more later.

**Comment by ultimatemonty** [comment]
@gschorkopf completely agree. Better to move now while we have very little to migrate over.
